### PR TITLE
Add EScAIP Model

### PR DESCRIPTION
**Pull request by EricZQu**
Hey maintainers,

Thanks for all the awesome work on this benchmark! I’m submitting our new model with this PR. Let me know if anything else is needed.

Please check the following items before submitting your PR:

- [x] I have created a new folder `models/<arch_name>/<model_variant>` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like training set names and/or important hyperparameters.
- [x] I have added the my new model as a new attribute on the [`Model.<arch_name>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) in `enums.py`.
- [x] I have uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<arch_name>-preds.csv.gz`).
- [ ] I have uploaded the model-relaxed structures file to Figshare or another cloud storage service (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.json.gz`).
- [ ] I have uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [ ] I have included a YAML metadata file (`models/<arch_name>/<model_variant>.yml`) with model details and the urls to the Figshare files. If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [ ] I have included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.

### CODE REVIEW

File: models/escaip/escaip.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+repo: https://github.com/ASK-Berkeley/EScAIP
+doi: https://doi.org/10.48550/arXiv.2410.24169
+url: https://github.com/ASK-Berkeley/EScAIP
+paper: https://openreview.net/forum?id=Y4mBaZu4vy
+pr_url: https://github.com/janosh/matbench-discovery/pull/191
+
+requirements:
+  fairchem-core: 1.2.1
+
+openness: OSCD
```
Comment (CompRhys) [inline models/escaip/escaip.yml]: labelling the model OSCD [Open source, closed data] is incompatible with it just being trained on MPtrj below

Comment (janosh) [reply models/escaip/escaip.yml]: good catch, this was a copy paste error on my part (i remember copying this from MACE-MPA-0). will add a test to catch this in future. probably each dataset in [`training-sets.yml`](https://github.com/janosh/matbench-discovery/blob/fd1dda6c11b20a941c352955a89a20b333f6ec5d/data/training-sets.yml#L36) should have an openness key as well

**Comment by janosh** [comment]
closing this as stale. feel free to reopen if there's interest to resume
